presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -526,6 +526,18 @@ You can also drop statistics for selected partitions only::
 
     CALL system.drop_stats(schema_name, table_name, ARRAY[ARRAY['p2_value1', 'p2_value2']])
 
+Delaying execution for dynamic filters
+---------------------------------------
+
+It can often be beneficial to wait for collection of
+:doc:`dynamic filters </admin/dynamic-filtering>` before starting table scan.
+This extra wait time can potentially result in significant overall savings
+in query and CPU time if dynamic filtering is able to reduce amount of scanned data.
+For the Hive connector, table scan can be delayed for a configured amount of time
+for the arrival of dynamic filters using the configuration property
+``hive.dynamic-filtering-probe-blocking-timeout`` or the session property
```

Comment:
I assume this is a catalog session property and actually just called dynamic_filtering_probe_blocking_timeout ... 

and hive is just the catalog name in this case .. we should clarify

Reply:
modified this to make it clearer